chain/actors/policy/policy.go.template

```diff
@@ -132,6 +132,17 @@ func GetMaxProveCommitDuration(ver actors.Version, t abi.RegisteredSealProof) ab
 	}
 }
 
+func SetProviderCollateralSupplyTarget(num, denom big.Int) {
+{{range .versions}}
+	{{if (ge . 2)}}
```

Comment:
Guessing we're not doing this for earlier versions just because it isn't that helpful and the `ProvCollateralPercentSupplyNumV0` / `ProvCollateralPercentSupplyNumV1` thing is messy?

Reply:
We didn't have supply target in v0 actors

Reply:
(and v1 actors didn't exist)

Reply:
isn't that what `ProvCollateralPercentSupplyNumV0` and `ProvCollateralPercentSupplyNumV1` did in v0 actors?

Reply:
Oh, maybe, but somehow those don't seem to matter